### PR TITLE
BT 5098 Metrics Cleanup for Feature Env

### PR DESCRIPTION
**Pull request by Shanmugapriya03**

### CODE REVIEW

**Review by raj-dhamsaniya — APPROVED** [review]
LGTM. @ajaychoudhary-hotstar we will use the same WORKING_DIRECTORY var when for some repo, unit test needed to run from diff dir.

File: .github/workflows/hotstar-go-ci-workflow.yml [hunk 1]
```diff
@@ -309,22 +304,22 @@ jobs:
...
           . fetch_config secret/non-prod/qe/testenvironment
-          export APP_NAME=${{ steps.app-detail.outputs.new_app_name }}
+          export REPO_NAME=${GITHUB_REPO_FULL_NAME##*/}
           $(curl -XPOST -H 'Content-Type: application/json' -H 'Accept: */*' -H "Authorization: $Auth" -d '{
-                       "counter_name": "tes_kill_counter",
-                        "label_names": ["query","app_name"],
-                        "label_values": ["get_ondemand_prflow_kill","'"$APP_NAME"'"]
+                       "counter_name": "delete_feature_env_success_counter",
+                       "label_names": ["repo","branch","service"],
+                       "label_values": ["'"${{ GITHUB_REPO_FULL_NAME }}"'", "'"${{ branch }}"'", "'"${{ REPO_NAME }}"'"]
```
Comment (ajaychoudhary-hotstar) [inline .github/workflows/hotstar-go-ci-workflow.yml:311]: there is already one env variable "GITHUB_REF_NAME" which has a repo name without "origin/ref"
reference https://docs.github.com/en/actions/learn-github-actions/environment-variables

Comment (ajaychoudhary-hotstar) [reply .github/workflows/hotstar-go-ci-workflow.yml:311]: use same in all workflow.

Comment (Shanmugapriya03) [reply .github/workflows/hotstar-go-ci-workflow.yml:311]: image

Comment (ajaychoudhary-hotstar) [reply .github/workflows/hotstar-go-ci-workflow.yml:311]: got it.

**Review by ajaychoudhary-hotstar — APPROVED** [review]

**Pull request merged**